isolate_example/README.md

```diff
@@ -19,4 +37,4 @@ the sample, the best places to go are:
 * [StackOverflow](https://stackoverflow.com/questions/tagged/flutter)
 
 If you run into an issue with the sample itself, please file an issue
-in the [main Flutter repo](https://github.com/flutter/flutter/issues).
\ No newline at end of file
+in the [main Flutter repo](https://github.com/flutter/flutter/issues).
```

Comment:
Not sure why this line is lit up as a change. It may have lost or gained a line break.